### ISSUE TITLE
Render suggestions in iron-list

### ISSUE THREAD

**Issue by nomego** [ellipticaljs/paper-autocomplete#104]
When providing a lot of suggestions, the dropdown is very slow, I have to limit it to around 30 to avoid it being sluggish.
To work around this, I think moving to iron-list would enable unlimited suggestions.

**Comment by jhuesos** [comment]
How many elements are we talking about? I would argue that the whole point of autocomplete is to prevent displaying too many elements.

About your suggestion, my only concern of using iron-list is that it will increase the number of dependencies on this component and therefore increase bundle size. I would have to check how much bigger it will grow

**Comment by ndormont** [comment]
I agree with Jaime, the suggestion list should be kept under 10 (IMHO). I personally leave the default value to 5 suggestions max. It's not very user friendly to offer more than 10 suggestions to a user, in a suggestion list. More results should be displayed in a separate container/page.

**Comment by nomego** [comment]
Our UX disagrees, if the user suggestion selection is limited, the user needs to be informed that additional options are available (with a refined search).
If users are presented with too many suggestions, they will likely refine the search further.
Also we work with lists where options are very similar, so users might do a partial search, start sifting through suggestions, and optionally search further.

Since iron list would enable a complete result set without hampering performance,  it would be a far better approach than "showing 5/3219"

**Comment by jhuesos** [comment]
As I said, my main concern is I am not sure how much weight it will add to
the component to include iron-list.

Another thing is I was expecting to re-write the component as a Polymer2
only component at some point very soon because the code and the component
really need a good refactoring and API cleanup.

Question: are you on Polymer1? 2? planning to migrate to 2 soon?

**Comment by nomego** [comment]
We're still on 1.x but our 2.x branch is expected to be merged before 2018 :)
